src/index.tsx

```diff
@@ -1341,9 +1338,6 @@ export class App extends React.Component<any, AppState> {
                     draggingElement: null,
                   });
                 }
-
-                history.resumeRecording();
-                this.setState({});
```

Comment:
This fixes the double rendering. Note that we setState on the two branches above so this one is strictly unecessary